src/System.Diagnostics.PerformanceCounter/tests/System.Diagnostics.PerformanceCounter.Tests.csproj

```diff
@@ -15,5 +16,18 @@
     <Compile Include="$(CommonTestPath)\System\ShouldNotBeInvokedException.cs">
       <Link>Common\System\ShouldNotBeInvokedException.cs</Link>
     </Compile>
+    <Compile Include="PerformanceDataTests.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="$(CommonTestPath)\System\Diagnostics\RemoteExecutorConsoleApp\RemoteExecutorConsoleApp.csproj">
+      <Project>{69e46a6f-9966-45a5-8945-2559fe337827}</Project>
+      <Name>RemoteExecutorConsoleApp</Name>
+    </ProjectReference>
```

Comment:
@safern @ViktorHofer @danmosemsft thank's for help...seems to work now, however it's not clear to me how it worked on other legs without this references.

Reply:
The exception was thrown here: https://github.com/dotnet/corefx/blob/6ab836db8df5ec9af77063e45f42adccf86ec0b5/src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs#L59

The only two frameworks that TestHost points to RemoteExecutorConsoleApp.exe are uap and netfx, and you’re skipping the tests that use RemoteExecutorTestBase on uap.

Netcoreapp uses dotnet as the TestHost.

Reply:
The exception was thrown here: https://github.com/dotnet/corefx/blob/6ab836db8df5ec9af77063e45f42adccf86ec0b5/src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs#L59

The only two frameworks that TestHost points to RemoteExecutorConsoleApp.exe are uap and netfx, and you’re skipping the tests that use RemoteExecutorTestBase on uap.

Netcoreapp uses dotnet as the TestHost.

Reply:
Understood thank's a lot!